scripts/sql-test-integration.bat

```diff
@@ -4,14 +4,18 @@ pushd %~dp0\..
 
 set VSCODEUSERDATADIR=%TMP%\adsuser-%RANDOM%-%TIME:~6,5%
 set VSCODEEXTENSIONSDIR=%TMP%\adsext-%RANDOM%-%TIME:~6,5%
-set PYTHON_TEST_PATH=%VSCODEUSERDATADIR%\TestPythonInstallation
 echo %VSCODEUSERDATADIR%
 echo %VSCODEEXTENSIONSDIR%
-echo %PYTHON_TEST_PATH%
 @echo OFF
 
-:: This first notebook test will do the python install that the later tests depend on
-call .\scripts\code.bat --extensionDevelopmentPath=%~dp0\..\extensions\notebook --extensionTestsPath=%~dp0\..\extensions\notebook\out\integrationTest --user-data-dir=%VSCODEUSERDATADIR% --extensions-dir=%VSCODEEXTENSIONSDIR% --remote-debugging-port=9222
+if "%SKIP_PYTHON_INSTALL_TEST%" == "1" (
```

Comment:
I'm curious if there's any reason to put both 1 and the env var in quotes here for the check. Is there a benefit over not having quotes?

Reply:
If the variable isn't set, then the script breaks at that line without the quotes.

Reply:
cool, thanks